### PR TITLE
Llama2 70b host offload for qkv proj activations

### PR DESCRIPTION
**Pull request by yejingxin**
Tried out llama2 70b host offload with fdsp, we got

v5p-128 65% MFU
v5e-256 57% MFU

### CODE REVIEW

**Comment by yejingxin** [comment]
This PR is ready for review.

File: MaxText/layers/models.py [hunk 1]
```diff
@@ -222,7 +222,13 @@ def __call__(self,
             'query_proj', 'value_proj', 'key_proj'
         )
       elif cfg.remat_policy == 'minimal_offloaded':
-        policy = jax.checkpoint_policies.offload_dot_with_no_batch_dims(offload_src="device", offload_dst="pinned_host")
```
Comment (rwitten) [inline MaxText/layers/models.py:225]: nit: avoid meaningless whitespace changes.

**Review by rwitten — APPROVED** [review]
Approved with nit!

But we also need to document this for customers.

**Comment by yejingxin** [comment]
added `proj_offloaded` to config file.

**Comment by yejingxin** [comment]
duplicated with #546

**Pull request closed without merging**